carbonmark/components/pages/Project/Retire/Pool/RetireForm.tsx

```diff
@@ -335,6 +337,11 @@ export const RetireForm: FC<Props> = (props) => {
     }
   }, [retirementBlockNumber]);
 
+  const permittedPaymentMethods: Array<CarbonmarkPaymentMethod> =
+    props.project.registry.startsWith("ICR")
+      ? CarbonmarkPaymentMethods.filter((method) => method !== "fiat")
+      : Array.from(CarbonmarkPaymentMethods);
+
   return (
```

Comment:
Maybe add a comment here so that it's clear this is temporary until ICR fiat is enabled

Reply:
I added a todo list item in
https://github.com/KlimaDAO/klimadao/issues/2159